src/socketio.quiver.js

```diff
@@ -407,14 +457,31 @@ QuiverSocialProvider.prototype.sendAllRosterChanged_ = function() {
   }
 };
 
-QuiverSocialProvider.prototype.countOwnerConnections_ = function() {
-  var count = 0;
-  for (var serverKey in this.connections_) {
-    if (this.connections_[serverKey].owner) {
-      ++count;
+/**
+ * @return {!Array.<!QuiverSocialProvider.server_>} The intersection of servers
+ *     that I am connected to and servers that I am advertising in my intro
+ *     message.  Includes connections that have not yet succeeded.
```

Comment:
I'm not sure about this, but I think the current behavior might make sense.  Connection failures should be rare, and informing the other party early about which servers we will be on seems like it should reduce latency.  If the connection does fail, I don't think we lose anything.